src/commands.ts

```diff
@@ -66,6 +67,17 @@ export function executeSelection(repls: Map<string, vscode.Terminal>): void {
   });
 }
 
+export function openMacroStepper(repls: Map<string, vscode.Terminal>): void {
+  withEditor((editor: vscode.TextEditor) => {
+    withFilePath((filePath: string) => {
+      withREPL((command: string[]) => {
```

Comment:
I think it might be better to use `withRacket` instead of `withREPL` for this feature. When you use REPL, the REPL session is left open even after closing the macro stepper. If you use the "plain" Racket command, you can send your macro stepper instructions via several `--eval` options.

Reply:
That would need to escape characters in terminal, and I'm not sure if that's possible for all syntax, e.g.

```
racket --eval "(expand/step #'(define f (latex-lambda (x y) "x^2 + y^2")))"
```

got

```
string::26: read: expected a `)` to close `(`
```